utils/rpcd-mod-poe/files/poe.acl

```diff
@@ -0,0 +1,14 @@
+{
+	"poe": {
+		"description": "manipulate gpio pins to enable/disable poe",
+		"read": {
+			"ubus": {
+				"poe": [
+					"on",
```

Comment:
The `on` and `off` procedures have side effects and should not be part of the read scope. Declare a `write` scope and put `on` & `off` there while only keeping `status` in `"read": { ... }`